app/src/main/java/com/github/se/eventradar/ui/qrCode/MyQrCodeScreen.kt

```diff
@@ -0,0 +1,55 @@
+package com.github.se.eventradar.ui.qrCode
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.github.se.eventradar.R
+import com.github.se.eventradar.viewmodel.MyQrCodeViewModel
+
+@Composable
+fun MyQrCodeScreen(viewModel: MyQrCodeViewModel) {
+  LaunchedEffect(Unit) {
```

Comment:
can this go in the `init` block of the view model? Also, if both of these methods are only called once and at the beginning, why are there two methods?